README.md

```diff
@@ -38,6 +38,15 @@ Using any of the supported integration methods, streamed files will appear along
 
 Supported ways to enable the DagsHub Streaming include:
 
+## Data Streaming - Installation (**temporary**)
+
+Use Git to install the Data Streaming Client on your machine, by running the following command:
+
+```bash
+pip install git+https://github.com/DagsHub/client.git@alpha
+```
```

Comment:
Just edit the `# Installation` section and change the target to `git+https://github.com/DagsHub/client.git@alpha`